PowerShellTools/DebugEngine/BreakpointManager.cs

```diff
@@ -0,0 +1,240 @@
+﻿using log4net;
+using Microsoft.VisualStudio;
+using Microsoft.VisualStudio.Debugger.Interop;
+using PowerShellTools.Common.ServiceManagement.DebuggingContract;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation.Runspaces;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PowerShellTools.DebugEngine
+{
+    /// <summary>
+    /// Breakpoint manager to handle variaties of operations on breakpoint in powershell runspace
+    /// </summary>
+    public class BreakpointManager

```

Comment:
Does not seems to be simple change, as it is exposed through debugger to different places.